ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -104,37 +104,43 @@ interface State {
 }
 
 const css = `
+  .p-growl {
+    position: sticky;
+  }
   .p-growl.p-growl-topright {
-     height: 29px;
-     width: 7rem;
-     padding-top: 6rem;
-     line-hieght: 1rem;
+     height: 1rem;
+     width: 6.4rem;
+     padding-top: 0.4rem;
+     line-hieght: 0.7rem;
      margin-right: 3rem;
-
+   }
+   .p-growl .p-growl-item-container .p-growl-item .p-growl-image {
+     font-size: 1rem !important;
+     margin-top: 0.19rem
    }
 
   .p-growl-item-container::before {
```

Comment:
What does the `::before` do? I removed it and didn't notice any change in the display.